src/contrib/persistence/Akka.Persistence.Sql.Linq2Db/Journal/DAO/BaseByteArrayJournalDao.cs

```diff
@@ -0,0 +1,421 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Event;
+using Akka.Persistence.Sql.Linq2Db.Config;
+using Akka.Persistence.Sql.Linq2Db.Db;
+using Akka.Persistence.Sql.Linq2Db.Journal.Types;
+using Akka.Persistence.Sql.Linq2Db.Serialization;
+using Akka.Streams;
+using Akka.Streams.Dsl;
+using LanguageExt;
+using LinqToDB;
+using LinqToDB.Data;
+using static LanguageExt.Prelude;
+using Seq = LanguageExt.Seq;
+
+namespace Akka.Persistence.Sql.Linq2Db.Journal.DAO
+{
+    public abstract class BaseByteArrayJournalDao :
+        BaseJournalDaoWithReadMessages,
+        IJournalDaoWithUpdates
+    {
+
+        public ISourceQueueWithComplete<WriteQueueEntry> WriteQueue;
+        protected JournalConfig _journalConfig;
+        protected FlowPersistentReprSerializer<JournalRow> Serializer;
+
+        private Lazy<object> logWarnAboutLogicalDeletionDeprecation =
+            new Lazy<object>(() => { return new object(); },
```

Comment:
TODO: This is supposed to log and warn that logical delete is deprecated... but only do so once